### PR TITLE
feat: viro poc

### PR DESCRIPTION
**Pull request by brainbicycle**
This PR resolves [] 

### PR Checklist

- [ ] I tested my changes on **iOS** / **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

File: android/settings.gradle [hunk 1]
```diff
@@ -1,3 +1,9 @@
 rootProject.name = 'Artsy'
 apply from: file("../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesSettingsGradle(settings)
 include ':app'
+// required by viro: https://github.com/ViroCommunity/viro/blob/v2.21.1/readmes/INSTALL_ANDROID.md
+include ':react_viro', ':arcore_client', ':gvr_common', ':viro_renderer'
+project(':arcore_client').projectDir = new File('../node_modules/@viro-community/react-viro/android/arcore_client')
+project(':gvr_common').projectDir = new File('../node_modules/@viro-community/react-viro/android/gvr_common')
+project(':viro_renderer').projectDir = new File('../node_modules/@viro-community/react-viro/android/viro_renderer')
+project(':react_viro').projectDir = new File('../node_modules/@viro-community/react-viro/android/react_viro')
```
Comment (pvinis) [inline android/settings.gradle:9]:
```suggestion
project(':react_viro').projectDir = new File('../node_modules/@viro-community/react-viro/android/react_viro')

```

Comment (pvinis) [reply android/settings.gradle:9]: eof newline

File: android/app/src/main/java/net/artsy/app/ArtsyAndroidUtils.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package net.artsy.app;
+
+import android.os.Build;
+
+public class ArtsyAndroidUtils {
+  public static boolean IsRunningOnEmulator() {
```
Comment (pvinis) [inline android/app/src/main/java/net/artsy/app/ArtsyAndroidUtils.java:6]: wow! where did this come from? pretty intense, but seems nice!

Comment (brainbicycle) [reply android/app/src/main/java/net/artsy/app/ArtsyAndroidUtils.java:6]: https://github.com/viromedia/viro/issues/269 comment in this issue to workaround the crash

**Comment by brainbicycle** [comment]
Gonna close this one for now as we are gonna try another approach, can always come back to it.

**Pull request closed without merging**